test/conformance/ingressv2/util.go

```diff
@@ -1073,12 +1073,11 @@ func WaitForHTTPRouteState(ctx context.Context, client *test.GatewayAPIClients,
 func IsHTTPRouteReady(r *gatewayv1alpha1.HTTPRoute) (bool, error) {
 	for _, gw := range r.Status.Gateways {
 		for _, condition := range gw.Conditions {
-			if condition.Type == "true" && condition.Status == metav1.ConditionTrue {
-				return true, nil
+			if condition.Type == "Admitted" && condition.Status != metav1.ConditionTrue {
```

Comment:
Can we use https://github.com/kubernetes-sigs/gateway-api/blob/71f93116ac7c2f69500d194fcd3a908b3f74e1b9/apis/v1alpha1/shared_types.go#L175 to make this comparison?